pkgs/development/libraries/fmt/default.nix

```diff
@@ -11,8 +11,6 @@ stdenv.mkDerivation rec {
     sha256 = "1hl9s69a5ql5nckc0ifh2fzlgsgv1wsn6yhqkpnrhasqkhj0hgv4";
   };
 
-  outputs = [ "out" "dev" ];
```

Comment:
What is the reason for no longer splitting the derivation?

Reply:
Perhaps the installed CMake files point to wrong location? Check the files under `share/cmake` in the `dev` output.

Reply:
That was mostly a fix for the input to work. Definitely should fix this, when I can understand what is the correct build input so that the header files can be found.